onnxruntime/core/providers/acl/nn/pool.cc

```diff
@@ -145,7 +145,7 @@ Status Pool<T, PoolType>::Compute(OpKernelContext* context) const {
 
   const Tensor* X = context->Input<Tensor>(0);
 
-  std::vector<int64_t> dilations(PoolBase::pool_attrs_.dilations);
+  TensorShapeVector dilations(PoolBase::pool_attrs_.dilations);
   std::vector<int64_t> aclDilations(2);
```

Comment:
>std::vector<int64_t> [](http://example.com/codeflow?start=2&length=20)

InlinedVector

Reply:
Would this become `absl::InlinedVector<int64_t, 2> aclDilations;` then?

Reply:
onnxruntime::InlinedVector<int64_t>